runtime/src/lib.rs

```diff
@@ -399,6 +400,71 @@ impl pallet_scheduler::Config for Runtime {
     type Preimages = Preimage;
 }
 
+parameter_types! {
+    pub const TreasuryPalletId: PalletId = PalletId(*b"zk/trsry");
+    pub const ProposalBond: Permill = Permill::from_percent(5);
+    pub const ProposalBondMinimum: Balance = 2000 * CENTS;
+    pub const ProposalBondMaximum: Balance = THOUSANDS;
+    pub const SpendPeriod: BlockNumber = 6 * DAYS;
+    pub const Burn: Permill = Permill::from_percent(1);
```

Comment:
I'm not necessarily asking for a change here, but I'd like to draw attention to this.

Do we really want to commit to spending Treasury funds promptly? I'm wondering whether it might be better to set this to `0`, at least initially, until we have established practices for handling these funds.

Reply:
Ok will check this with John

Reply:
@drgora checked with John, so

- Changed burn to 0 as requested (will change that 1 or something else later but not for now) 
- No changed SpendPeriod: His worry was that treasury would pay no matter what but that is not case, every 6 days fund will be paid only for approved proposals and if sufficient funds are available.

Reply:
This is exactly what I had in mind, thanks!